doc/source/projections.rst

```diff
@@ -64,11 +64,17 @@ Be aware that **most of the time** you don't have to set a projection. Data load
 
 From time to time, however, you may get data that does not include a projection. In this situation, you have to set the CRS so *geopandas* knows how to interpret the coordinates.
 
-For example, if you convert a spreadsheet of latitudes and longitudes into a GeoSeries by hand, you would set the projection by assigning the WGS84 latitude-longitude CRS to the :attr:`GeoSeries.crs` attribute:
+For example, if you convert a spreadsheet of latitudes and longitudes into a
+GeoSeries by hand, you would set the projection by assigning the WGS84
+latitude-longitude CRS to the :attr:`GeoSeries.crs` attribute or by passing the
+CRS to the :meth:`GeoSeries.set_crs` method. The following approaches are
+all equivalent:
 
 .. sourcecode:: python
 
-   my_geoseries.crs = "EPSG:4326"
+    my_geoseries.crs = "EPSG:4326"
```

Comment:
I would remove this and all use cases of directly setting crs from docs (that could be another PR), because it skips CRS mismatch check we now have in `set_crs`. We should encourage using `set_crs` instead. It may also be confusing in the beginning that after setting `"EPSG:4326"`, `df.crs` returns pyproj CRS instead of original string.